src/global_scheduler/global_scheduler.c

```diff
@@ -136,19 +144,27 @@ void process_new_db_client(db_client_id db_client_id,
     plasma_photon_entry->aux_address = strdup(aux_address);
     plasma_photon_entry->photon_db_client_id = db_client_id;
     HASH_ADD_KEYPTR(
-        hh, state->plasma_photon_map, plasma_photon_entry->aux_address,
+        plasma_photon_hh, state->plasma_photon_map, plasma_photon_entry->aux_address,
         strlen(plasma_photon_entry->aux_address), plasma_photon_entry);
 
+    /* Add photon_db_client_id -> plasma_manager ip:port association to state.*/
+    HASH_ADD(photon_plasma_hh, state->photon_plasma_map,
+             photon_db_client_id, /* Key is the field name of entry struct. */
```

Comment:
We've been always putting comments on their own lines and not on separate lines.